server/src/services/standups.service.ts

```
@@ -169,33 +165,42 @@ export const deleteStandup = async (id: string) => {
     throw error;
   }
 };
+
 export const getStandup = async (id: string) => {
-  const standup = await prisma.standup.findFirst({
-    where: {
-      id: Number(id),
-    },
-    include: { questions: true, members: true, channel: true },
-  });
+  try {
+    const standup = await prisma.standup.findFirst({
+      where: {
+        id: Number(id),
+      },
+      include: { questions: true, members: true, channel: true },
+    });
+    console.log(standup);
+    
 
-  let modifiedStandup = {
-    id: standup?.id,
-    title: standup?.title,
-    originalTime: standup?.originalTime,
-    days: standup?.days,
-    questions: standup?.questions,
-    channel: {
-      id: standup?.channel[0].slackChannelId,
-      name: standup?.channel[0].name,
-    },
-    members: standup?.members.map((member) => {
-      return {
-        id: member.slackMemberId,
-        name: member.name,
-      };
-    }),
-  };
+    let modifiedStandup = {
```

remove optional chanining here & also as mentioned last time put `TODO` tag for this (as we need to handle this in FE)

---

Done, however removing optional chaining will break it as we are not handling it in FE yet

---

Can you explaing how removing optional chaining breaks and how not handling in FE is related to this ?

---

If we remove the optional chaining, we get an error that "Object is possibly null"
Typescript thinks our standup property can be null, hence, it won't let us. The solution for it uses conditions, optional chaining, or disabling strict null checks.

---

You can throw an error if standup not found instead of using optional chaining